config/environments/profile.rb

```diff
@@ -0,0 +1,41 @@
+# an environment configured for performance profiling.
```

Comment:
Here's the line from our environments/production.rb which I think is the one you were asking about, and is not here. 
https://github.com/chemheritage/chf-sufia/blob/master/config/environments/production.rb#L27